newIDE/app/src/ProjectManager/ExtensionsPropertiesDialog.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import FlatButton from '../UI/FlatButton';
+import Dialog from '../UI/Dialog';
+import Text from '../UI/Text';
+import { ColumnStackLayout } from '../UI/Layout';
+import EmptyMessage from '../UI/EmptyMessage';
+import PropertiesEditor from '../PropertiesEditor';
+import propertiesMapToSchema from '../PropertiesEditor/PropertiesMapToSchema';
+import { List } from '@material-ui/core';
+
+type Props = {|
+  project: gdProject,
+  open: boolean,
+  onApply: Function,
+|};
+
+class ExtensionsPropertiesDialog extends React.Component<Props> {
```

Comment:
Can we make this *not* a dialog, but just a component that you use inside `ProjectPropertiesDialog`? So that the existing Ad Mob property, and the new ones, will be all shown like they are part of the dialog :)

![image](https://user-images.githubusercontent.com/1280130/90408677-6f05cc80-e0a8-11ea-8f8f-c1151b88cee7.png)